### PR TITLE
[CBRD-23493] thread related parameters

### PR DESCRIPTION
**Pull request by ribram**
http://jira.cubrid.org/browse/CBRD-23493

this patch concludes new (not hidden) system parameters

### CODE REVIEW

File: en/admin/config.rst [hunk 1]
```diff
@@ -1601,6 +1611,64 @@ The following are parameters related to SQL statements and data types supported
...
+    \After closing a connection, the thread will wait the value of the \
+    \parameter expressed in seconds to be assigned a new connection. If no \
+    \connection is assigned and the wait time expires, the thread stops. \
+    \Another thread may be started the next time a connection comes. \
+    \If parameter value is **-1**, threads never stop. They sleep until \
+    \they are given a new assignment.
+
+**thread_worker_pooling**
+
+    If **thread_connection_pooling** parameter is true, all threads used for client requests execution are pooled on server boot.
```
Comment (razvanradulescu) [inline en/admin/config.rst]:
```suggestion
    If **thread_worker_pooling** parameter is true, all threads used for client requests work execution are pooled on server boot.
```

File: en/admin/config.rst [hunk 2]
```diff
@@ -1601,6 +1611,64 @@ The following are parameters related to SQL statements and data types supported
...
+
+**thread_worker_timeout_seconds**
+
+    **thread_worker_timeout_seconds** is a parameter that configures \
+    \wait time before stopping for threads handling client requests. \
```
Comment (razvanradulescu) [inline en/admin/config.rst]:
```suggestion
    \wait time before stopping for threads handling work execution. \
```

File: en/admin/config.rst [hunk 3]
```diff
@@ -1601,6 +1611,64 @@ The following are parameters related to SQL statements and data types supported
...
+    \After executing a request, the thread will wait the value of the \
```
Comment (razvanradulescu) [inline en/admin/config.rst]: Now, the worker threads only handle client requests work, but in we may push other work which is not originated by client. What about reformulating to give a more generic meaning ?

Comment (ribram) [reply en/admin/config.rst]: I am afraid "work" would be too generic and thus without real meaning.

File: en/admin/config.rst [hunk 4]
```diff
@@ -1601,6 +1611,64 @@ The following are parameters related to SQL statements and data types supported
       
         1, NULL
 
+.. _thread-parameters:
+
+Thread-Related Parameters
+-------------------------
+
+Thread managemenet can be confirued by threads parameters. The type and value range for each parameter are as follows:
```
Comment (eseokoh) [inline en/admin/config.rst]:
```suggestion
Thread management can be configured by threads parameters. The type and value range for each parameter are as follows:
```

File: en/admin/config.rst (hunk 3 above)
Comment (eseokoh) [reply en/admin/config.rst]: I support "client request:".

File: en/admin/config.rst (hunk 1 above)
Comment (eseokoh) [inline en/admin/config.rst]:
```suggestion
    If **thread_worker_pooling** parameter is true, all threads used for client requests execution are pooled on server boot.
```

File: en/admin/config.rst (hunk 3 above)
Comment (ribram) [reply en/admin/config.rst]: Ok, let's keep client request for now. If in the future pool is changed to do something else, we can think of better naming.

**Pull request merged**